info.yml

```diff
@@ -22,7 +22,15 @@ display_name: "Desktop Startup Framework"
 description: "Startup logic for the Shotgun Desktop"
 
 # Required minimum versions for this item to run
-requires_core_version: "v0.19.3"
+# Do not update this core version, or the upgrade of the Shotgun Desktop
```

Comment:
I have not modified this file. Github is high. If you diff it with the master branch, it's the same. I had create this branch on the wrong commit apparently.